docker/cron/backup/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Google LLC
```

Comment:
Should we just have a single "cron" Dockerfile that can execute all the cron jobs?

Reply:
yeah, I agree with it. I will create a base cron job docker and override the entrypoint in each k8s yaml.